buildSrc/src/main/java/com/google/firebase/gradle/plugins/DackkaPlugin.kt

```diff
@@ -204,26 +204,26 @@ abstract class DackkaPlugin : Plugin<Project> {
   private fun GenerateDocumentationTask.applyCommonConfigurations() {
     dependsOnAndMustRunAfter("createFullJarRelease")
 
-    val dackkaFile = project.provider { project.dackkaConfig.singleFile }
+    val dackkaFile = project.layout.projectDirectory.file(project.dackkaConfig.singleFile.path)
 
-    dackkaJarFile.set(dackkaFile)
+    dackkaJarFile.convention(dackkaFile)
     clientName.set(project.firebaseLibrary.artifactId)
   }
 
   private fun registerFiresiteTransformTask(
     project: Project,
-    dackkaOutputDirectory: Provider<File>,
+    dackkaOutputDirectory: Provider<RegularFile>,
   ) =
     project.tasks.register<FiresiteTransformTask>("firesiteTransform") {
       val outputDir by tempFile("dackkaTransformedFiles")
 
-      dackkaFiles.set(dackkaOutputDirectory.childFile("docs/reference"))
+      dackkaFiles.set(dackkaOutputDirectory.get().asFile.childFile("docs/reference"))
```

Comment:
`.get()` eagerly resolves the provider. You'll wanna `map` it instead.

Reply:
done